dynamic_values/config_modules/clusters/clusters.tf

```diff
@@ -62,6 +62,7 @@ module "composed_cluster_map" {
       vpc_id           = var.vpc_modules[cluster.vpc_name].vpc_id
       subnets          = module.cluster_subnets[cluster.name].subnets
       cos_instance_crn = cluster.kube_type == "openshift" ? var.cos_instance_ids[cluster.cos_name] : null
+      access_tags      = []
```

Comment:
After looking at this I actually think this scenario is different from the other two as this is merging in external information into the object of the cluster that is needed to create it so I've actually removed the access tags from this because they will already be in the cluster object itself. I tried going with the `cluster.access_tags` but that failed the plan saying cluster does not have an attribute of access_tags.